browser.js

```diff
@@ -255,7 +255,7 @@ Requests.type = 'XHR' === Requests.method ? (function detect() {
     type = types.pop();
 
     xhr = Requests.XHR();
-    xhr.open('get', '/', true);
+    xhr.open('get', 'http://localhost/', true);
```

Comment:
Wouldn't that cause security issues because it's now testing against a users local host?